### PR TITLE
disable LTO for sys-libs/libomp

### PR DESCRIPTION
**Pull request by Titaniumtown**
Title: sys-libs/libomp: disable LTO

Linking failed for libomp-11.0.0_rc4 with LTO, it works now.

### CODE REVIEW

File: sys-config/ltoize/files/package.cflags/ltoworkarounds.conf [hunk 1]
```diff
@@ -266,8 +267,7 @@ net-libs/libtorrent-rasterbar *FLAGS-="-mtls-dialect=gnu2" # causes memory corru
 # END: TLS dialect workarounds
 
 # BEGIN: Semantic Interposition workarounds
-app-crypt/tpm2-tss *FLAGS-="${SEMINTERPOS}" # Problem with inlining, see issue #592
-app-misc/tracker *FLAGS-="${SEMINTERPOS}" # builds but makes gnome-base/nautilus deadlock
```
Comment (telans) [inline sys-config/ltoize/files/package.cflags/ltoworkarounds.conf:270]: I think you missed removing the above comment

Comment (Titaniumtown) [reply sys-config/ltoize/files/package.cflags/ltoworkarounds.conf:270]: oops, idk why those 2 things got deleted, let me fix that rq.

Comment (Titaniumtown) [reply sys-config/ltoize/files/package.cflags/ltoworkarounds.conf]: Fixed!

**Comment by aruhier** [comment]
Linked to issue #619.

**Comment by Titaniumtown** [comment]
@InBetweenNames Could this please be merged? I don't see any reason for it not to be.

**Comment by InBetweenNames** [comment]
Sorry -- I've been especially busy lately.  This looks good to me, and I'll go ahead and mark the LLVM packages as no-LTO as well.

**Pull request merged**